doc/documentation_contributors.md

```diff
@@ -0,0 +1,116 @@
+Guideline for contributing documentation
+========================================
+The DLite documentation is written in Markdown.
+This include both the README files and documentation found in the `doc/` subdirectory.
+
+Common to both is that the text should be as easy and natural as possible to read from the terminal.
+Hence, the following recommendations should be followed:
+
+* Write one sentence per line, in order to get an easier to read output from `git diff`.
+
+* For README files, use the underline style format for main and sub-headers.
+  For example, start a new page with
+
+      Overview
+      ========
+
+  instead of
+
+      # Overview
+
+  For documentation the style is up to the writer.
+
+* Add two newlines before headers to make them easier to recognise.
+
+  This is especially useful for levels 3-headers and below (which are lines starting with 3 or more hashes (`#`)) to make them easier to recognise as sub-headers.
+
+* Avoid the use of html tags.
+
+* Links become more readable if you place them at the end of the document using square brackets.
+  Example:
+
+     ```
+     A link to [SOFT].
+     ...
+
+     End of document.
+
+
+     [SOFT]: https://www.sintef.no/en/publications/publication/1553408/
+     ```
+
+* Use `.md` as file extension for all Markdown files.
+
+The README files are intended to document the overall project or the content in a subdirectory.
+These should therefore render nicely on GitHub.
+Hence, use [Basic Markdown] or possible [GitHub-flavored Markdown].
+Keep the maximum line length at 79 characters so that the source file easily can be viewed in a 80 character wide terminal.
+
+
+The Markdown files in the `doc/` subdirectory are intended to be included in the sphinx-generated [online documentation].
+Here we can make use of the [MyST Markdown extensions], like [admonitions].
+
+
+Figures
+-------
+Please place figures in the `doc/figs/` directory.
+If you use [drawio], it is recommended that you save the figure in svg format.
+Then it renders well in browsers and is easy to find and edit for collaborators.
+
+
+Examples
+--------
+Write your python examples in the same way as done in the [official Python documentation], by preceeding expressions with `>>> ` and the expected output on the next line below the `>>> `.
+For example
+
+    ```python
+        # Here is a python example
+        >>> x = 1 + 1
+        >>> x
+        2
+
+    ```
+This allow to check your examples by running
+
+    python -m doctest <filename>
+
+on your markdown file.
+
+It is recommended to indent your code with 4 spaces.
+That makes it easy to copy the correct output into your example.
+For instance, if you write
+
+    ```python
+        >>> 1 + 1  # doctest: +SKIP
+        3
+
+    ```
+
```

Comment:
```suggestion
```
This should not be an issue and is something that is handled by the Sphinx theme.

Reply:
It is, because the `doctest` output is written to the terminal.